configs/config-1.31/files/kubeadm-default.conf

```diff
@@ -0,0 +1,42 @@
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: InitConfiguration
+certificateKey: 34a7a99924b249364a53f0cd2af07893c39549f3fa5ad0efafb01b193f7868fe
+nodeRegistration:
+  kubeletExtraArgs:
+    tls-min-version: VersionTLS12
+    address: 0.0.0.0
+    authorization-mode: AlwaysAllow
+---
+apiVersion: kubeadm.k8s.io/v1beta3
+kind: ClusterConfiguration
+apiServer:
+  extraArgs:
+    tls-min-version: VersionTLS12
+controllerManager:
+  extraArgs:
+    tls-min-version: VersionTLS12
+    bind-address: 0.0.0.0
+scheduler:
+  extraArgs:
+    tls-min-version: VersionTLS12
+    bind-address: 0.0.0.0
+networking:
+  serviceSubnet: 10.96.0.0/12
+  podSubnet: 10.244.0.0/16
+imageRepository: container-registry.oracle.com/olcne
+kubernetesVersion: 1.30.3
+etcd:
+  local:
+    imageRepository: container-registry.oracle.com/olcne
+    imageTag: 3.5.12
```

Comment:
Could this be updated to 3.5.15?

Reply:
Updated to 3.5.18, which is what is going out.  There were some hard to fix problems with 3.5.15.